DocumentsFromSnapshotMigration/docker/runJavaWithClasspath.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec java -XX:MaxRAMPercentage=80.0 -cp /rfs-app/jars/*:. "$@"
```

Comment:
this is a file that we were automatically generating w/in the gradle file - are we still doing that?  Is this a good time to transition over to the gradle-managed application scripts that it builds by default for the java applications?